lib/core/src/lib/language-menu/language-picker.component.spec.ts

```diff
@@ -0,0 +1,61 @@
+/*!
+ * @license
+ * Copyright © 2005-2025 Hyland Software, Inc. and its affiliates. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { LanguagePickerComponent } from './language-picker.component';
+import { MatMenuItem, MatMenuTrigger } from '@angular/material/menu';
+import { LanguageMenuComponent } from './language-menu.component';
+import { QueryList } from '@angular/core';
+import { CoreTestingModule, UnitTestingUtils } from '@alfresco/adf-core';
+
+describe('LanguagePickerComponent', () => {
+    let component: LanguagePickerComponent;
+    let fixture: ComponentFixture<LanguagePickerComponent>;
+    let testingUtils: UnitTestingUtils;
+
+    beforeEach(async () => {
+        await TestBed.configureTestingModule({
+            imports: [CoreTestingModule, LanguagePickerComponent]
+        }).compileComponents();
+    });
+
+    beforeEach(() => {
```

Comment:
Do we need two separate beforeEach? Can't we combine this one with the one above? Usually we have single beforeEach per describe

Reply:
done